database/spanner/README.md

```diff
@@ -2,14 +2,16 @@
 
 ## Usage
 
-The DSN must be given in the following format.
+See [Google Spanner Documentation](https://cloud.google.com/spanner/docs) for
+more details.
 
-`spanner://projects/{projectId}/instances/{instanceId}/databases/{databaseName}`
+The DSN must be given in the following format.
 
-See [Google Spanner Documentation](https://cloud.google.com/spanner/docs) for details.
+`spanner://projects/{projectId}/instances/{instanceId}/databases/{databaseName}?option=true`
 
+as described in [README.md#database-urls](../../README.md#database-urls)
 
-| Param | WithInstance Config | Description |
+| Option | WithInstance Config | Description |
```

Comment:
Leave this as `Param` since these are URL query params: https://golang.org/pkg/net/url/#Values

Reply:
Hi - can do  -what was confusing to me about that is that 
https://github.com/golang-migrate/migrate/#database-urls uses `option=`. Would it make sense to change that to param=?

Reply:
See if it makes more sense this way. I assumed that the example in the main readme would be the one to follow, but this makes sense, I think?

Reply:
Oh yeah, good catch. Let's be consistent and also change the docs in the repo root's README to use params instead of options. Thanks!